### PR TITLE
Allow Analytical platform access to a viper subdirectory of the Risk …

### PR DESCRIPTION
**Pull request by Mjwillis**
…Profiler S3 bucket.

### CODE REVIEW

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/categorisation-tool-production/resources/s3.tf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module "risk_profiler_digcat_s3_bucket" {
-  source                 = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=2.0"
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=bucket_policy"
```
Comment (Mjwillis) [inline namespaces/cloud-platform-live-0.k8s.integration.dsd.io/categorisation-tool-production/resources/s3.tf]: didn't know if this should now be changed to ref=2.0 (depends if merged to master?)

Comment (alkar) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/categorisation-tool-production/resources/s3.tf]: Yes please, can we please hold off on this one until the PR is merged?

Comment (alkar) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/categorisation-tool-production/resources/s3.tf]: This should be version [3.0](https://github.com/ministryofjustice/cloud-platform-terraform-s3-bucket/releases/tag/3.0) now

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/categorisation-tool-dev/resources/s3.tf [hunk 2]
```diff
@@ -12,7 +12,7 @@ module "risk_profiler_digcat_s3_bucket" {
 }
 
 module "viper_digcat_s3_bucket" {
-  source = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=bucket_policy"
+  source = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=2.0"
```
Comment (alkar) [inline namespaces/cloud-platform-live-0.k8s.integration.dsd.io/categorisation-tool-dev/resources/s3.tf]: You might want to update this to 3.0 as well, for consistency.

Comment (Mjwillis) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/categorisation-tool-dev/resources/s3.tf]: done

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/categorisation-tool-production/resources/s3.tf [hunk 3]
```diff
@@ -1,5 +1,5 @@
 module "risk_profiler_digcat_s3_bucket" {
-  source                 = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=2.0"
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=3.0"
```
Comment (Mjwillis) [inline namespaces/cloud-platform-live-0.k8s.integration.dsd.io/categorisation-tool-production/resources/s3.tf]: moved to 3.0

**Pull request merged**